src/consensus/tower.zig

```diff
@@ -89,28 +92,48 @@ pub const Tower = struct {
         fork_root: Slot,
         slot_account_reader: sig.accounts_db.SlotAccountReader,
     ) !void {
+        self.logger.info().logf(
+            "initializeLockoutsFromBank: fork_root={}, vote_pubkey={}",
+            .{ fork_root, vote_account_pubkey },
+        );
         const vote_account = blk: {
-            const maybe_vote_account = slot_account_reader.get(
+            const maybe_vote_account = try slot_account_reader.get(
                 allocator,
                 vote_account_pubkey.*,
-            ) catch |err| switch (err) {
-                error.OutOfMemory => |e| return e,
-                error.InvalidOffset,
-                error.FileIdNotFound,
-                error.SlotNotFound,
-                => null,
-            };
+            );
             break :blk maybe_vote_account orelse {
+                self.logger.warn().logf(
```

Comment:
isn't this a normal situation for a new validator? why would it be a warning message?